src/actions/bitcoinActions.js

```diff
@@ -234,7 +234,7 @@ const bitcoinWalletCreationFailed = (): BitcoinWalletCreationFailedAction => ({
 
 export const initializeBitcoinWalletAction = (wallet: EthereumWallet) => {
```

Comment:
Should we change this type as well? Maybe something like `KeychainWalletData = { mnemonic?: string, privateKey: string };`

Reply:
it will be in a different set of work, maybe an optimization Friday ? :)

Reply:
actually there is no need to change this type :) I thought you're getting the data straight from the keychain, but was wrong as it's the same `EthereumWallet` object as it was before